sqlmesh/core/audit/builtin.py

```diff
@@ -57,9 +57,9 @@
 number_of_rows_audit = ModelAudit(
     name="number_of_rows",
     query="""
-SELECT COUNT(*)
+SELECT COUNT(*) as _count
 FROM (
-   SELECT 1
+   SELECT 1 as _one
```

Comment:
I'm probably just out of the loop but is this not something we think SQLGlot should handle? I could see us forgetting this with future changes.